### PR TITLE
Emin/3

### PR DESCRIPTION
**Pull request by eminaydin**
Apparently it was third exercise :smiley_cat:

### CODE REVIEW

File: routes/middleware/firstTask.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let myFunc = function (req, res, next) {
+    console.log(req.method +  " on " + req.originalUrl );
+    console.log(req.body);
+    
+    
+    next();
+}
```
Comment (oliverwebr) [inline routes/middleware/firstTask.js]: This is exercise 2. Pull requests should tackle one problem at a time

File: routes/students.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const express = require("express");
+const secondTask = require("./middleware/secondTask");
```
Comment (oliverwebr) [inline routes/students.js]: secondTask variable makes no sense in the context of the server. :)

**Comment by oliverwebr** [comment]
@eminaydin you need to push the changes, there are no new commit here

**Comment by eminaydin** [comment]
I've pushed them already to emin/3.

**Comment by oliverwebr** [comment]
@eminaydin accidentally closed or will create a new pull request?

**Comment by eminaydin** [comment]
I'll create a new one in second.

**Comment by oliverwebr** [comment]
Alright :) good luck 🤞

**Pull request closed without merging**